packages/core/src/command/patient-import/patient-import-shared.ts

```diff
@@ -15,30 +15,54 @@ const globalPrefix = "patient-import";
 
 export type FileStages = "raw" | "valid" | "invalid";
 
-function createCxJobPrefix(cxId: string, jobId: string): string {
-  return `cxid=${cxId}/jobid=${jobId}`;
+function createCxJobPrefix(cxId: string, jobStartedAt: string, jobId: string): string {
+  return `cxid=${cxId}/date=${jobStartedAt.slice(0, 10)}/jobid=${jobId}`;
```

Comment:
Fine for this v0, but I'd validate this is a valid ISO date in a followup PR.

Reply:
Couldn't we just use the current date? Does it need to absolutely match the day at which the job was really triggered (in the case its started close to midnight)?

Reply:
No -- we want all data under the same path and date is *before* job id.